client/src/main/java/org/apache/celeborn/client/read/WorkerPartitionReader.java

```diff
@@ -96,11 +96,12 @@ public void onFailure(int chunkIndex, Throwable e) {
             exception.set(new CelebornIOException(errorMsg, e));
           }
         };
-    TransportClient client;
+    TransportClient client = null;
     try {
       client = clientFactory.createClient(location.getHost(), location.getFetchPort());
     } catch (InterruptedException ie) {
-      throw new CelebornIOException("Interrupted when createClient", ie);
+      logger.error("PartitionReader thread interrupted while creating client.");
```

Comment:
When ```createReader``` throws InterruptedException, ```RssInputStream.createReaderWithRetry``` should not retry other partitions. Maybe we can fix this in later PR.